normandy/control/static/control/js/components/form_fields/CheckboxField.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const CheckboxField = (props) => {
```

Comment:
I'm not terribly excited about the `const Foo = (args) => { }` style of function declaration. This is a style nit though, so feel free to ignore this.